corehq/apps/adm/models.py

```diff
@@ -348,8 +349,10 @@ def calculate_totals(self, column_data):
     def get_couch_view_data(self, key, datespan=None):
         default_value = None
         try:
-            now = tz_utils.adjust_datetime_to_timezone(getattr(datespan, self.start_or_end or "enddate"),
-                from_tz=datespan.timezone, to_tz=pytz.utc)
+            now = UserTime(
+                getattr(datespan, self.start_or_end or "enddate"),
+                datespan.timezone
+            ).server_time().done().replace(tzinfo=pytz.utc)
```

Comment:
Yeah this `done()` thing is something I have mixed feelings about. The original idea is that the these *Time helper classes converted only between each other, and then to get a datetime back out, you have to explicitly call `.done()`. (So in this case, you call `.done()` to get a datetime and then call `datetime.replace` on it. (This code uses timezones sloppily; I believe this is here because the output of `dateutil.parse` was timezone aware and the author of it just wanted these things to be comparable to each other, which was kind of par for the course of how we were treating timezones 3 years ago.)

Anyway, this line is "right" but I'm not crazy about needing to call done on every function, so I'll probably change this api a bit in the near future.